_docs/cardano/2017-01-13-wallet.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Wallets
+permalink: /cardano/wallets/
+group: cardano
+---
+[//]: # (Reviewed at e74b95fd7e04b43c03198dbed0f8599d53df5235)
+
+# Wallets in Cardano SL
+
+While addresses discussed in [Addresses](/cardano/addresses/) section, are
+fundamental to send and receive funds, wallets are a way to simplify these
+processes for end-users.
+
+## What is a Wallet?
+
+In Cardano, wallets are defined in the following manner:
+
+~~~ haskell
```

Comment:
You're right, the *Wallets* doc in its current form should go into *Technical details* entirely. However, in that case I should probably remove the *Cardano SL Node* section and replace it with the old *Wallets* doc, migrated to the *Technical details* section. What do you think?